webgl.js

```diff
@@ -666,14 +666,6 @@ gl.getContextAttributes = function () {
   return this._contextattributes
 }
 
-gl.getSupportedExtensions = function getSupportedExtensions () {
```

Comment:
Is the removal of `getSupportedExtensions` correct? Should it be kept in to only return the supported extensions (including a check for AngleInstanceArraysSupport)?